### ISSUE TITLE
Error logs advise: “ should use self.async_write_ha_state()”

### ISSUE THREAD

**Issue by dredjohn21** [uvjim/linksys_velop#424]
Hi

Noticed this in the logs.  I’m guessing something changed with the HAOS code?  Everything is working perfectly…

`Logger: homeassistant.helpers.entity
Source: helpers/entity.py:560 
First occurred: May 5, 2023 at 11:39:10 PM (3 occurrences) 
Last logged: May 6, 2023 at 2:16:20 PM

Entity device_tracker.phone (<class 'custom_components.linksys_velop.device_tracker.LinksysVelopMeshDeviceTracker'>) is using self.async_update_ha_state(), without enabling force_update. Instead it should use self.async_write_ha_state(), please report it to the custom integration author.
Entity device_tracker.android (<class 'custom_components.linksys_velop.device_tracker.LinksysVelopMeshDeviceTracker'>) is using self.async_update_ha_state(), without enabling force_update. Instead it should use self.async_write_ha_state(), please report it to the custom integration author.
Entity device_tracker.watch (<class 'custom_components.linksys_velop.device_tracker.LinksysVelopMeshDeviceTracker'>) is using self.async_update_ha_state(), without enabling force_update. Instead it should use self.async_write_ha_state(), please report it to the custom integration author.
`

**Comment by uvjim** [comment]
Yup. Looks like something has changed. I didn't get a chance to run the beta this time.

Looks like it was introduced [here](https://github.com/home-assistant/core/pull/91372).

It doesn't look like it causes an issue but I'll get it fixed.

Thanks for reporting.

**Comment by uvjim** [comment]
The fix should be in `2023.5.1b0`

**Comment by dredjohn21** [comment]
Wow you are fast - thanks so much!!

**Comment by uvjim** [comment]
Sorry just pushed a 2nd beta because there were errors when changing state with the last fix.
